exercises/sublist/example.rb

```diff
@@ -0,0 +1,18 @@
+
+def sublist(set_a, set_b)
```

Comment:
There are actually great methods around doing this in the Ruby standard library - maybe you could find some inspiration there for a more easily readable example implementation? http://ruby-doc.org/stdlib-2.3.1/libdoc/set/rdoc/Set.html